DigitalLearningSolutions.Web/Styles/shared/searchableElements/filter.scss

```diff
@@ -113,7 +113,3 @@
     margin-right: 4px;
   }
 }
-
-.card-filter-tag {
-  @extend .card-tag;
-}

```

Comment:
I had some problems with this file when both tags and common were being used. Specifically, it could not extend the .input-with-submit-button, but when the tags file was removed, it worked fine. Seeing as how this card-filter-tag was the only thing using it, I moved this to the tags file instead.